InventoryBuilder.py

```diff
@@ -175,6 +179,7 @@ def query_inventory_permanent(self):
             self.get_datastores()
             self.get_vms()
             self.get_nsxt_mgmt_cluster()
+            self.get_nsxt_mgmt_nodes()
```

Comment:
Maybe not for this PR, but something to discuss:

With growing complexity I find a bit confusing to have 
`Vrops.get_nsxt_mgmt_nodes
self.get_nsxt_mgmt_nodes`

Which pretty much looks the same, but ain't. Going with a standard approach here would be to rather use `self._get_nsxt_mgmt_nodes` instead.

What's your opinion? Also, if we do the big renaming bulk, it should go in a separate PR.

Reply:
changed all functions from `get_...` to `expose_...`